src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Ssl.cs

```diff
@@ -289,6 +268,10 @@ protected override bool ReleaseHandle()
         private SafeSslHandle() : base(IntPtr.Zero, true)
         {
         }
+
+        internal SafeSslHandle(IntPtr validSslPointer) : base(validSslPointer, false)
```

Comment:
Having the default be false here is likely going to cause problems in the future, when someone unsuspectingly creates a SafeSslHandle around an IntPtr expecting the SafeSslHandle to own its lifetime.  We should have this ctor also take the Boolean value and force the call site to provide it.